.prettierrc.yaml

```
@@ -1,9 +1,11 @@
+arrowParens: avoid
 semi: false
 singleQuote: true
+trailingComma: none
```

Personally I agree with the Prettier defaults here, but for the sake of touching fewer files added the overrides to preserve this repo's old behavior. Thoughts?

```suggestion
semi: false
singleQuote: true
```

---

Ye, keeping the output as close to what we have in the repo currently is preferable (although I also agree that we could switch defaults of those 2 later on)